subprojects/performance/templates.gradle

```diff
@@ -27,7 +27,7 @@ tasks.register("gradleBuildBaseline", RemoteProject) {
     remoteUri = rootDir.absolutePath
     // Remember to update accordingly when rebasing/squashing
     // Do not use the "Rebase and merge" nor "Squash and merge" Github buttons when merging a PR that change the baseline
-    ref = 'b1f9b6ff5ce6b00f96f49faf486345409de97ae4'
+    ref = '066b239ee28ae063f2deb1a9de02ad394c4b6e21'
```

Comment:
You need to bump the baseline so that the performance tasks are fixed?

Reply:
Yes, exactly (cf. commit message of dfb8b2f58b82a9bb73f19ed601043b9386e2f36c).